src/client/App.js

```diff
@@ -96,6 +96,7 @@ export function wrapComponentInProvider(Comp, initialState = {}) {
  * @param {String} target - The html id of the target you want to render into.
  */
 export function renderComponent(Comp, target) {
+  setInterval(() => checkForModeratorMessages(), 30000);
```

Comment:
Bør der være et tjek for om man er logget ind. Der er vil ingen grund til at lave disse kald for anonyme brugere.  

checkForModeratorMessages bør også kaldes en gang med det samme. Lige nu går det 30 sekunder for vi kalder første gang